spec-main/api-app-spec.ts

```diff
@@ -240,9 +240,9 @@ describe('app module', () => {
       expect(code1).to.equal(0);
       const data2 = (await data2Promise)[0].toString('ascii');
       const secondInstanceArgsReceived: string[] = JSON.parse(data2.toString('ascii'));
-      const expected = process.platform === 'win32'
-        ? [process.execPath, '--some-switch', '--allow-file-access-from-files', appPath, 'some-arg']
-        : secondInstanceArgs;
+      const expected = process.platform === 'darwin'
+        ? [process.execPath, '--some-switch', '--allow-file-access-from-files', '--enable-avfoundation', appPath, 'some-arg']
```

Comment:
This flag comes from `./shell/app/electron_main_delegate.cc`

Reply:
These flags will keep changing depending on the chromium version, it is not a good idea to keep updating this spec. What this spec wants to check if custom command line args are passed back to the first instance, so I suggest to only check for presence of those flags in the final result.

Reply:
I noticed that the flags differed between the different platforms being tested below, and ended up with the same thought. Pushing up a changed test